debezium-server-core/pom.xml

```diff
@@ -168,10 +178,11 @@
                 <configuration>
                     <skipTests>${skipITs}</skipTests>
                     <enableAssertions>true</enableAssertions>
-                    <systemProperties>
+                    <systemPropertyVariables>
                         <test.type>IT</test.type>
                         <test.apicurio>false</test.apicurio>
-                    </systemProperties>
+                        <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+                    </systemPropertyVariables>
```

Comment:
This is the problematic part. 

Prior to this change, integration tests don't run in IntelliJ idea  as it seems that the following error in IT results in no further output (and it looks like the test was not executed)

```
log4j:WARN No appenders could be found for logger (org.jboss.logging).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Sep 02, 2024 4:03:52 PM org.jboss.logmanager.JBossLoggerFinder getLogger
ERROR: The LogManager accessed before the "java.util.logging.manager" system property was set to "org.jboss.logmanager.LogManager". Results may be unexpected.
```

Reply:
@jpechane  any opinion regarding this?

Reply:
@jcechace If it works in CLI then I've no issue with it.